src/commands/install.yml

```diff
@@ -88,6 +93,16 @@ steps:
                   exit 1
               esac
 
+              # If `less` is not installed and AWS_PAGER is not set explicitly
+              # override with the empty string so that certain commands do not fail.
+
+              if [ "<<parameters.disable-aws-pager>>" == "true" && -z "$(command -v less)" && -z "${AWS_PAGER}"]; then
```

Comment:
I agree, I think it'd be good to disable paging regardless of whether `less` is installed, because paging would likely break builds as well whenever manual intervention is required.